app/actions/ControlActions.js

```diff
@@ -1045,6 +1051,7 @@ export const LOCKACCOUNT_SUCCESS = "LOCKACCOUNT_SUCCESS";
 export const unlockAcctAndExecFn = (
   passphrase,
   acctNumber,
+  secondaryAcctUnlock,
```

Comment:
I think we can make acctNumber an array, and unlock all sent. This way would avoid needing a third param, in case. What do you think?

Reply:
yah i think i like the sounds of that, will change